src/Symfony/Component/DomCrawler/Form.php

```diff
@@ -282,6 +282,11 @@ public function get(string $name)
         return $this->fields->get($name);
     }
 
+    public function &getReference(string $name)
```

Comment:
`getReference` or `getByReference`?

Reply:
I just thought that maybe we could have new method that returns collections by reference and it would be more understandable? `public function &getCollection(string $name)`

Reply:
getCollection, and we force the return type to `array`?

Reply:
Exactly. We can throw properly before returning: `if (!\is_array(...`